### ISSUE TITLE
variable glitch

### ISSUE THREAD

**Issue by Djidjey** [TurboWarp/scratch-gui#1059]
The "coordonnées x" variable changes its value depending on the sprite, while the variable is public. Huh?

https://user-images.githubusercontent.com/93743322/158066501-ea7fedc9-141e-403b-9a0f-aae8e13714d5.mp4

**Comment by Mickmick21** [comment]
"turbowarp editor cannot use cloud variables"so it is not cummunicating to the turbowarp server, if you want you can give to me the project to try to fix the problem
Français(puisque je vois que c'est en français): "L'éditeur turbowarp ne peut pas utiliser les variables clouds"donc cela veut dire que l'éditeur ne communique pas aux server turbowarp, si tu veux, tu peux me donner ton project pour essayer de régler le problème.

**Comment by Mickmick21** [comment]
et "money" est pour un seul joueur ou  pour tous ? car les variables clouds sont pour tous le monde

**Comment by GarboMuffin** [comment]
The variable they're having an issue with is not a cloud variable

**Comment by Mickmick21** [comment]
yes i know but It doesn't use Turbowarp servers so even with normal variables they cannot be public (unless it use turbowarp servers)
